django/forms/formsets.py

```diff
@@ -297,6 +296,10 @@ def is_valid(self):
         """
         if not self.is_bound:
             return False
+        if not self.management_form.is_valid():
+            error = _('Management form data is missing or has been tampered with.')
```

Comment:
This should be overridable by `default_error_message` and passing `error_messages` to the formset for consistency with `Field`.